test/provider-styles.test.js

```diff
@@ -8,16 +8,16 @@ import Component from './helpers/Component';
 import { wait, getWrapper, getMount } from './helpers/utils';
 
 describe('Styles', function () {
-
-  it('should load default stylesheet', () => {
+  it('should not load default stylesheet', () => {
```

Comment:
I updated this test to match the new functionality of not defaulting to rendering `AccountUnavailableComponent` until we attempt to fetch accounts.